config/locales/en.yml

```diff
@@ -1,64 +1,69 @@
-# Files in the config/locales directory are used for internationalization
```

Comment:
Having locales sorted is good, but it must be done from the start, not in the middle of the project:
Consequences of your change:
- Breaking git blame
- Causing potential conflicts for your teammates, that would otherwise be avoided

Will merge for now.
For the future: remember that you're not *alone* on the project.
The less lines you change the less conflicts there would be.
Cost of this change is *not* justified.